extensions/toggl-track/src/toggl/index.ts

```diff
@@ -1,72 +1,72 @@
-import { Client, Project, Tag, TimeEntry, Workspace } from "./types";
+import { Client, Me, Project, Tag, TimeEntry, Workspace } from "./types";
 import { getPreferenceValues } from "@raycast/api";
 import { authenticatedFetch } from "./auth";
 
 const TogglAPI = function (apiToken: string) {
-  const baseUrl = "https://api.track.toggl.com/api/v8";
+  const baseUrl = "https://api.track.toggl.com/api/v9";
   const api = authenticatedFetch(apiToken, baseUrl);
 
   return {
+    getMe: (): Promise<Me> => {
+      return api.get<Me>("/me");
+    },
     getWorkspaces: (): Promise<Workspace[]> => {
       return api.get<Workspace[]>("/workspaces");
     },
     getWorkspaceProjects: async (workspaceId: number): Promise<Project[] | null> => {
       const projects = (await api.get<Project[] | null>(`/workspaces/${workspaceId}/projects`)) || [];
       projects.push({
         id: -1,
```

Comment:
Could we change this to `undefined` as the api changed?

Reply:
If I understand correctly `-1` is used here to identify the "no project" item and it doesn't have anything to do with the API itself. If we changed it to `undefined` then we'd have to guard against it in all places this field is used. I think the current approach makes more sense.